[2025-01-22T23:26:49.743+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: retrieve_data_from_web.print_date_task manual__2025-01-22T23:26:44.712761+00:00 [queued]>
[2025-01-22T23:26:49.757+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: retrieve_data_from_web.print_date_task manual__2025-01-22T23:26:44.712761+00:00 [queued]>
[2025-01-22T23:26:49.758+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-22T23:26:49.784+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): print_date_task> on 2025-01-22 23:26:44.712761+00:00
[2025-01-22T23:26:49.793+0000] {standard_task_runner.py:60} INFO - Started process 5199 to run task
[2025-01-22T23:26:49.802+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'retrieve_data_from_web', 'print_date_task', 'manual__2025-01-22T23:26:44.712761+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/task1_retrieve_data.py', '--cfg-path', '/tmp/tmp7fixmb3v']
[2025-01-22T23:26:49.806+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask print_date_task
[2025-01-22T23:26:49.836+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-22T23:26:49.912+0000] {task_command.py:423} INFO - Running <TaskInstance: retrieve_data_from_web.print_date_task manual__2025-01-22T23:26:44.712761+00:00 [running]> on host a8cdec4ae828
[2025-01-22T23:26:50.061+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='retrieve_data_from_web' AIRFLOW_CTX_TASK_ID='print_date_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T23:26:44.712761+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-22T23:26:44.712761+00:00'
[2025-01-22T23:26:50.625+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-22T23:26:50.641+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=retrieve_data_from_web, task_id=print_date_task, execution_date=20250122T232644, start_date=20250122T232649, end_date=20250122T232650
[2025-01-22T23:26:50.702+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-22T23:26:50.733+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
